app/Http/Requests/UserSessionRequest.php

```diff
@@ -0,0 +1,69 @@
+<?php namespace App\Http\Requests;
+
+use Candidate;
+use Input;
+use Auth;
+
+class UserSessionRequest extends Request
+{
+
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return Auth::guest();
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $rules = [
+            'first_name' => 'required',
+            'phone' => 'required_without:email|regex:/^((1)?([\-\s\.]{1})?)?\(?([0-9]{3})\)?(?:[\-\s\.]{1})?([0-9]{3})(?:[\-\s\.]{1})?([0-9]{4})/',
+            'email' => 'required_without:phone|email',
+            'birthdate' => 'required|date|before:today',
+        ];
+
+        return $rules;
+    }
+
+    /**
+     * Set custom messages for validator errors.
+     *
+     * @return array
+     */
+    public function messages()
+    {
+        return [
+            'first_name.required' => 'What\'s your name?!',
+            'phone.required_without' => 'Give us your digits!',
+            'phone.regex' => 'That doesn\'t look like a real phone number!',
+            'email.required_without' => 'We need your email',
+            'email.email' => 'We need a valid email',
+            'birthdate.required' => 'When were you born?',
+            'birthdate.date' => 'Enter your birthday MM/DD/YYYY!',
```

Comment:
Yep, it's in the `$rules` variable a few lines up. Not sure what the default error message is, but it'll use that unless we override here.